config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,23 @@
+redis_url = "#{ENV["REDIS_URL"]}/0"
```

Comment:
I think `REDIS_URL` default to database `0` if not specified.  As the database number can be set as part of the environment variable, I suspect this might cause problems - i.e. `redis://example.com:6379/5/0`

Reply:
👍